backend/getData.js

```diff
@@ -27,7 +27,11 @@ function getData(element: Object): DataType {
   var text = null;
   var publicInstance = null;
   var nodeType = 'Native';
-  if (element._currentElement === null || element._currentElement === false) {
+  // Sometimes element is just a string or number
+  if (typeof element !== "object") {
```

Comment:
It is... most of the time. Just not when you've got a string element of an `<option/>`, which is what has been causing #248. In that case, and only in that case, `element._renderedChildren` is a list of raw strings/numbers, b/c it's not allowed to put a `<span>` inside of an `<option>`. In every other case, if there are multiple children, then `element._renderedChildren` is a list of more elements.